### PR TITLE
Replace EmptyGraph with AnonymousTraversalSource if possible

### LINKED ISSUE #1563: Use AnonymousTraversalSource to create a TraversalSource
The new static method `traversal()` is the recommended way to create a `GraphTraversalSource` [since TinkerPop 3.3.5](http://tinkerpop.apache.org/docs/3.4.0/upgrade/#_anonymoustraversalsource). We should change our docs accordingly and all places where we use other methods.

### PR DESCRIPTION
**Pull request by farodin91**
Fixes #1563

Signed-off-by: Jan Jansen <jan.jansen@gdata.de>

-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [x] Is there an issue associated with this PR? Is it referenced in the commit message?
- [x] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [x] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [x] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](https://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[doc only]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

File: docs/connecting/java.md [hunk 1]
```diff
@@ -67,8 +67,7 @@ To get started with JanusGraph in Java:
 4.  Create a `GraphTraversalSource` which is the basis for all Gremlin traversals:
 
     ```java
-    Graph graph = EmptyGraph.instance();
-    GraphTraversalSource g = graph.traversal().withRemote("conf/remote-graph.properties");
+    GraphTraversalSource g = AnonymousTraversalSource.traversal().withRemote("conf/remote-graph.properties");
```
Comment (FlorianHockmann) [inline docs/connecting/java.md:70]: Please don't use the `AnonymousTraversalSource` directly. The idea was to create it with just `traversal()` which requires to statically import `AnonymousTraversalSource`. See [this section in the TinkerPop reference docs](http://tinkerpop.apache.org/docs/current/reference/#connecting-gremlin-server) where it's shown for the different languages or [in the upgrade docs](http://tinkerpop.apache.org/docs/current/upgrade/#_anonymoustraversalsource) where it was explained when it was introduced in TinkerPop.

Comment (farodin91) [reply docs/connecting/java.md]: I basically update all use cases again for this one. Do you like to check it again?

File: docs/basics/multi-node.md [hunk 2]
```diff
@@ -145,9 +145,7 @@ local reference to a remote graph:
 ```groovy
 gremlin> cluster = Cluster.open('conf/remote-objects.yaml')
 ==>localhost/127.0.0.1:8182
-gremlin> graph = EmptyGraph.instance()
-==>emptygraph[empty]
-gremlin> g = graph.traversal().withRemote(DriverRemoteConnection.using(cluster, "graph1_traversal"))
+gremlin> g = AnonymousTraversalSource.traversal().withRemote(DriverRemoteConnection.using(cluster, "graph1_traversal"))
```
Comment (FlorianHockmann) [inline docs/basics/multi-node.md:148]: Still with `AnonymousTraversalSource.traversal()` here

Comment (farodin91) [reply docs/basics/multi-node.md:148]: In this case i wasn’t sure how write it correctly.

Comment (FlorianHockmann) [reply docs/basics/multi-node.md:148]: What do you mean? Did you try it out in the console? The console already has the static imports. So it just works:

```groovy
gremlin> cluster = Cluster.open('conf/remote-objects.yaml')
==>localhost/127.0.0.1:8182
gremlin> g = traversal().withRemote(DriverRemoteConnection.using(cluster, "graph1_traversal"))
==>graphtraversalsource[emptygraph[empty], standard]
```

Comment (farodin91) [reply docs/basics/multi-node.md]: Do you like to review it again?

**Pull request merged**